### ISSUE TITLE
Regression for blur effect in Firefox

### ISSUE THREAD

**Issue by eeshaan** [Niels-IO/next-image-export-optimizer#67]
https://github.com/Niels-IO/next-image-export-optimizer/blob/78435e719aec4606551c14033b1dff11dcd53c14/src/ExportedImage.tsx#L132

The above line results in the blur effect not applying because the URL `#sharpBlur` is not found.

**Comment by Niels-IO** [comment]
Hi @eeshaan,

Thanks for submitting. The bug is harder to fix than expected. I found out that Firefox will not apply the blur filter as it has display:none applied. 
Changing to position: "absolute",height: 0 works instead.

Somehow the blurred placeholder image will not show up completely, only the section of the image that is loaded already:
<img width="678" alt="image" src="https://user-images.githubusercontent.com/47059882/198225679-160ab09c-2f61-4ec4-8b58-4456f90c5681.png">
I tried to search for it and haven't found a viable solution. Interestingly, if you stop all network requests at that point, the picture shows and change a property of the img element, and Firefox re-renders to its full height.

https://user-images.githubusercontent.com/47059882/198227131-a5582d72-669f-4c8c-8cab-e8c7b22db170.mov
